README.md

```diff
@@ -77,15 +77,21 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
 
         conda config --env --add channels usgs-astrogeology
 
+        # If you have followed conda-forges advice to set `channel_priority=strict`,
+        # you need to set it to flexible for the install:
+        conda config --set channel_priority flexible
+
 
 6.  The environment is now ready to download ISIS and its dependencies:
 
-        conda install -c usgs-astrogeology isis
+        # To install e.g. version 4.3
```

Comment:
I'd move this up into the text above.

The environment is now ready to download ISIS and its dependencies. To install the latest release, version 4.3, run:

    conda install isis=4.3